contracts/modules/assessment/Assessment.sol

```diff
@@ -153,6 +153,7 @@ contract Assessment is IAssessment, MasterAwareV2 {
   ///                membership during stake lockup period and thus allowing the user to withdraw
   ///                their staked amount to the new address when possible.
   function unstake(uint96 amount, address to) external whenNotPaused override {
+    require(block.timestamp > nxm.isLockedForMV(msg.sender), "Assessment: Nxm locked for voting");
```

Comment:
I think we can relax this requirement a bit. If the member has voted in governance require the `to` to be `msg.sender`. What do you think?

Reply:
Also please make the token symbol uppercase - `NXM` (it's already uppercased in staking).